hazelcast-client/src/main/java/com/hazelcast/client/util/ClientDelegatingFuture.java

```diff
@@ -38,20 +39,14 @@
  */
 public class ClientDelegatingFuture<V> implements InternalCompletableFuture<V> {
 
+    private static final Object NULL_OBJECT = new Object();
```

Comment:
I would call this VOID. Because null could afaik be a perfect legal return value.